app/connectors/itsastatus/ITSAStatusConnector.scala

```diff
@@ -88,24 +90,24 @@ class ITSAStatusConnector @Inject()(val http: HttpClient,
   def requestOptOutForTaxYear(taxableEntityId: String, optOutUpdateRequest: OptOutUpdateRequest)
                              (implicit headerCarrier: HeaderCarrier): Future[OptOutUpdateResponse] = {
 
-    http.PUT[OptOutUpdateRequest, HttpResponse](
-      url = buildUpdateRequestUrlWith(taxableEntityId),
-      body = optOutUpdateRequest,
-      headers = appConfig.getIFHeaders("2149")
-    ).map { response =>
-      val correlationId = response.headers.get(CorrelationIdHeader).map(_.head).getOrElse(s"Unknown_$CorrelationIdHeader")
-      response.status match {
-        case Status.NO_CONTENT => OptOutUpdateResponseSuccess(correlationId)
-        case _ =>
-          response.json.validate[OptOutUpdateResponseFailure].fold(
-            invalid => {
-              val msg = s"Json validation error parsing itsa-status update response, error $invalid"
-              logger.error(msg)
-              OptOutUpdateResponseFailure.defaultFailure(msg, correlationId)
-            },
-            valid => valid.copy(correlationId = correlationId, statusCode = response.status)
-          )
+    http.put(url"${buildUpdateRequestUrlWith(taxableEntityId)}")
+      .withBody(Json.toJson[OptOutUpdateRequest](optOutUpdateRequest))
+      .setHeader(headers2149: _*)
+      .execute[HttpResponse]
+      .map{ response =>
+        val correlationId = response.headers.get(CorrelationIdHeader).map(_.head).getOrElse(s"Unknown_$CorrelationIdHeader")
+        response.status match {
+          case NO_CONTENT => OptOutUpdateResponseSuccess(correlationId)
+          case _ =>
+            response.json.validate[OptOutUpdateResponseFailure].fold(
+              invalid => {
+                val msg = s"Json validation error parsing itsa-status update response, error $invalid"
+                logger.error(msg)
+                OptOutUpdateResponseFailure.defaultFailure(msg, correlationId)
+              },
+              valid => valid.copy(correlationId = correlationId, statusCode = response.status)
```

Comment:
also add logging for success, so this would be aligned with the way we do logging in _getITSAStatus_ method